ui/perfherder/alerts/StatusDropdown.jsx

```diff
@@ -197,6 +263,43 @@ export default class StatusDropdown extends React.Component {
             }
           />
         )}
+        <AlertModal
+          showModal={showCulpritBugModal}
+          toggle={() => this.toggle('showCulpritBugModal')}
+          updateAndClose={(event, inputValue) =>
+            this.fileBugAndClose(
+              event,
+              {
+                bug_number: parseInt(inputValue, 10),
+                issue_tracker: issueTrackers.find(
+                  (item) => item.text === selectedValue,
+                ).id,
+              },
+              'showCulpritBugModal',
+            )
+          }
+          header="Culprit Bug"
+          title="Bug Number"
+          submitButtonText="File Bug"
+          regression
```

Comment:
I don't understand why the purpose for this `regression` flag.

Reply:
The regression flag is `true` when the AlertModal is instanced by the `File bug` button. In `AlertModal.jsx:95` checks for the `regression` flag and show the button `File bug` with it's functionality of filing bug. If the `regression` flag is `false` (`AlertModal.jsx:108`), then it will show the other button (`Link to bug`). I wasn't sure whether this modifications worth creating another `AlertModal` file.

Reply:
Yeah. It seems we're bloating the `AlertModal` component because we need some "extra custom" behavior.
If a similar feature will show up in the future (can easily happen), we'll bloat it even more.

We need to ensure this component respects the open/closed principle. Currently, it didn't.

Reply:
I don't see any other option. I need to tell `AlertModal` somehow that this is a dialog for filing a regression, not for linking it. There's the option of creating a new `AlertModal` file, but it might be too much duplication since we only need to customize the behavior of the submit button. Looking into the future, we might want to improve this functionality of filing a regression, so probably I'm going to create a separate modal.